### DISCUSSION TITLE
Gitlab + Coolify

### DISCUSSION THREAD

**Discussion by deniz-itmatter** [coollabsio/coolify#4083] [Q&A]
Is there any way in v4 like the old Versions to use a self hosted gitlab as a normal source?

What are the settings for this 
![image](https://github.com/user-attachments/assets/c83d5db8-4de8-4242-8e41-c6e4bda97d05)

I always get this error

![image](https://github.com/user-attachments/assets/f7d245f6-5978-4da8-9b51-658b19b85dc7)

**Comment by peteryusuf** [comment]
I think a rewrite for the quide is needed to fully cover the scope of implementing gitlab hosted on coolify to a coolify instance. There's alot of confusion. Perhaps the guide will help you with your query however i am still stuck when i follow it. I feel it could be due the port its trying to connect to. 
[https://coolify.io/docs/knowledge-base/git/gitlab/integration](url)
